### PR TITLE
update for 17.0

### PR DESCRIPTION
**Pull request by OnGle**

### CODE REVIEW

File: overlay/usr/lib/inithooks/firstboot.d/20regen-mayan-secrets [hunk 1]
```diff
@@ -3,30 +3,41 @@
...
 sed -i "\|PASSWORD|s|:.*|:'$DB_PASS',|" $ENV
 
 # Update the DB user password in postgres
 $INITHOOKS_PATH/bin/pgsqlconf.py --user=$DB_USER --pass="$DB_PASS"
 
 # regen the Django secret
-PYTHONPATH=/opt/mayan-edms/lib/python3.7/site-packages $INITHOOKS_PATH/bin/mayan-gen-secret.py
+#. /opt/mayan-edms/bin/activate
+#python $INITHOOKS_PATH/bin/mayan-gen-secret.py
+#deactivate
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/firstboot.d/20regen-mayan-secrets]: Is the Django secret(s) regenerated some other way? Or did you just forget to uncomment it?

Regardless, if it's no longer needed, we should just remove it IMO.

**Comment by JedMeister** [comment]
Great work on this one! Looks like it was a super complex!

I've posted a comment against lines 25-27 of [`overlay/usr/lib/inithooks/firstboot.d/20regen-mayan-secrets`](https://github.com/turnkeylinux-apps/mayan-edms/pull/5/files#diff-1507b1cab25382dda6524317380597d655499b2172495daa5e69df7855155190) but I'm mentioning it here, just in case it's only visible to me... The lines are commented out and I just wanted to check that that is intentional (or perhaps you just forget to uncomment them after commenting them out for troubleshooting?

**Review by JedMeister — COMMENTED** [review]
Doh, I think I may not have published this previously?!

[update] Or maybe I did?! I wish the GH "review" system was a bit more clear about what is public and what isn't!

**Comment by OnGle** [comment]
Hmm, odd I'll take a look.

**Comment by OnGle** [comment]
No idea why I commented it out, but fix was easy enough.

**Pull request merged**